frontend/src/scenes/dashboard/dashboardLogic.tsx

```diff
@@ -97,7 +100,7 @@ export const dashboardLogic = kea<dashboardLogicType<DashboardLogicProps>>({
                             ? `api/shared_dashboards/${props.shareToken}`
                             : `api/projects/${teamLogic.values.currentTeamId}/dashboards/${props.id}/?${toParams({
                                   refresh,
-                                  dive_source_id,
+                                  dive_source_id: dive_source_id ? await getInsightId(dive_source_id) : undefined,
```

Comment:
Similar comment as another one below, but would prefer if this `getInsightId` logic is baked into the backend GET logic

Reply:
The dive dashboards are still behind a feature flag, and will need some work before we release them. I'd prefer to revisit this then. This variable is undefined most of the time now.

Reply:
Also, I changed this `id` to a `short_id` to be compatible with the logic here: https://github.com/PostHog/posthog/blob/6a64698f85d349cc55ca08c4589b17160640be36/frontend/src/scenes/dashboard/dashboardLogic.tsx#L300 -- both of these should be with the same type.